flixel/input/gamepad/id/FlxGamepadMotionValueList.hx

```diff
@@ -0,0 +1,29 @@
+package flixel.input.gamepad.id;
+import flixel.input.gamepad.FlxGamepad;
+
+/**
+ * A helper class for gamepad input.
+ * Provides optimized gamepad button checking using direct array access.
+ */
+@:keep
+class FlxGamepadMotionValueList
+{
+	private var gamepad:FlxGamepad;
+	
+	@:allow(flixel.input.gamepad.FlxGamepad)
+	public var available(default, null):Bool = true;
```

Comment:
No, it gets set to false if the gamepad model is switched to something that doesn't support motion.